### PR TITLE
Document getting tokens for API v2 using the client credentials flow

### PR DESCRIPTION
**Pull request by angelaLazar**
https://trello.com/c/hDkOVvwO/20-getting-tokens-for-api-v2-using-the-client-credentials-flow

### CODE REVIEW

File: articles/api/management/v2/tokens.md [hunk 1]
```diff
@@ -11,6 +11,52 @@ The Auth0 Management APIv2 token is required to call v2 of the Auth0 Management
 
 ## How to get a Management APIv2 Token
 
+### Using the Client Credentials Flow
+
+**Create a Non Interactive Client**
+
+A Non Interactive Client is a client that interacts with an API where there is no user involved. It's a machine to machine interaction. This must be used instead of a Single Page or Native apps because those cannot meet the necessary security requirements for executing this type of flow.
+
+To create a new Non Interactive Client, go to the [Clients section](${manage_url}/#/clients) of the dashboard then click the **CREATE CLIENT** button.
```
Comment (mpaktiti) [inline articles/api/management/v2/tokens.md:20]: In order for this to work the *Account Settings > Advanced > Enable APIs Section* has to be enabled. If it is not, then I do not see the *APIs* menu on the left and I do not have the *Non Interactive Client* option when I create a new Client. You can check [in this doc](https://auth0.com/docs/apis#how-to-configure-an-api-in-auth0).

File: articles/api/management/v2/tokens.md [hunk 2]
```diff
@@ -11,6 +11,52 @@ The Auth0 Management APIv2 token is required to call v2 of the Auth0 Management
...
+
+Enter a name for your new client, select **Non Interactive Clients** and then click the **CREATE** button.
+
+![](/media/articles/api/tokens/noninteractive-client.png)
```
Comment (mpaktiti) [inline articles/api/management/v2/tokens.md]: Missing alternate text.

File: articles/api/management/v2/tokens.md [hunk 3]
```diff
@@ -11,6 +11,52 @@ The Auth0 Management APIv2 token is required to call v2 of the Auth0 Management
...
+
+**Authorize the Client**
+
+After creating the new client, you will be brought to it's **Quick Start** section. Select **Auth0 Management API** from the dropdown. You will then see a message that the client is not authorized, click **NAVIGATE TO THE API AND AUTHORIZE**.
+
+![](/media/articles/api/tokens/navigate-button.png)
```
Comment (mpaktiti) [inline articles/api/management/v2/tokens.md:30]: Missing alternate text.

File: articles/api/management/v2/tokens.md [hunk 4]
```diff
@@ -11,6 +11,52 @@ The Auth0 Management APIv2 token is required to call v2 of the Auth0 Management
...
+
+This will bring you to the [APIs section](${manage_url}/#/apis) under **Auth0 Management API** in the **Non Interactive Clients** section.
+
+![](/media/articles/api/tokens/authorize-noninteractive.png)
```
Comment (mpaktiti) [inline articles/api/management/v2/tokens.md]: Missing alternate text.

File: articles/api/management/v2/tokens.md [hunk 5]
```diff
@@ -11,6 +11,52 @@ The Auth0 Management APIv2 token is required to call v2 of the Auth0 Management
...
+
+Toggle the slider to authorize your client.
+
+**Choose the scopes**
+
+You will see the available scopes that you can be granted for this client (which can be revoked at any time). Choose the desired scopes and then click the **UPDATE** button.
+
+![](/media/articles/api/tokens/choose-scopes.png)
```
Comment (mpaktiti) [inline articles/api/management/v2/tokens.md]: Missing alternate text.

File: articles/api/management/v2/tokens.md [hunk 6]
```diff
@@ -11,6 +11,52 @@ The Auth0 Management APIv2 token is required to call v2 of the Auth0 Management
...
+
+**Getting the token**
+
+To test the client interaction with the API, go to the **Test** section under **Auth0 Management API**.
+
+![](/media/articles/api/tokens/test-client.png)
```
Comment (mpaktiti) [inline articles/api/management/v2/tokens.md]: Missing alternate text.

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey Angela, I left some feedback. In addition to that, the screenshots seem wrong, they do not have the *APIs* menu on the left.

**Comment by angelaLazar** [comment]
@mpaktiti I addressed your comments in the latest commit

**Review by mpaktiti — APPROVED** [review]

**Pull request merged**